src/services/software-modernisation/index.html

```diff
@@ -10,9 +10,9 @@
 redirect_from: /software-modernisation/
 
 intro:
-  background-image-small: /assets/custom/img/softmod/bg-hero-dark.jpg
-  background-image-large: /assets/custom/img/softmod/bg-hero-light.jpg
-  foreground-image-large: /assets/custom/img/softmod/img-hero.png
+  background-image-small: /assets/custom/img/softmod/software-modernisation-hero-small-med.jpg
+  background-image-large: /assets/custom/img/softmod/software-moderisation-hero-large.jpg
```

Comment:
We started moving images out of the `/img/softmod/` and into either `/img/services/software-modernisation/` or `/img/services/` depending on which page they're on. Should we move these ones into `img/services/software-modernisation/` to reflect where they are used?

Reply:
This is now in effect from this commit: [0c9adfb](https://github.com/codurance/site/pull/1867/commits/0c9adfb62ef79d312a00ffb0e60e7b2cf0d17745)